chart/templates/mayastor/obs/stats-deployment.yaml

```
@@ -0,0 +1,49 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-stats
+  labels:
+    app: stats
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stats
```

```suggestion
      app: obs-callhome-stats
```
also on the other ones